packages/bsky/src/api/app/bsky/graph/getTrendingTopics.ts

```diff
@@ -0,0 +1,97 @@
+import { noUndefinedVals } from '@atproto/common'
+import AppContext from '../../../../context'
+import { Server } from '../../../../lexicon'
+import {
+  createPipeline,
+  HydrationFnInput,
+  PresentationFnInput,
+  RulesFnInput,
+  SkeletonFnInput,
+} from '../../../../pipeline'
+import { HydrateCtx, Hydrator } from '../../../../hydration/hydrator'
+import { Views } from '../../../../views'
+import { QueryParams } from '../../../../lexicon/types/app/bsky/unspecced/getTrendingTopics'
+import AtpAgent from '@atproto/api'
+import { TrendingTopic } from '../../../../lexicon/types/app/bsky/unspecced/defs'
+import { InternalServerError } from '@atproto/xrpc-server'
+
+export default function (server: Server, ctx: AppContext) {
+  const getTrendingTopics = createPipeline(
+    skeleton,
+    hydration,
+    noBlocksOrMutes,
+    presentation,
+  )
+  server.app.bsky.unspecced.getTrendingTopics({
+    auth: ctx.authVerifier.standard,
```

Comment:
oh, do we want this to be able to be hit unauthed? if so, we can use `standardOptional` here

Reply:
ah nice, ty